### PR TITLE
Push Tutorial to Production

### PR DESCRIPTION
**Pull request by Madeline-Schaefer**

### CODE REVIEW

File: tutorials/private-link-aws/private-link-aws.md [hunk 1]
```diff
@@ -9,19 +9,20 @@ primary_tag: software-product-function>sap-btp-cockpit
...
 
 ## Prerequisites
- - You have a global account and subaccount on SAP Business Technology Platform with SAP Private Link service entitlement: [Set Up SAP Private Link Service on Amazon Web Services (Beta)](private-link-service-onboarding-aws).
- - You have created an AWS Private Link Service in the Amazon VPC Console. [Create a service powered by AWS Private Link](https://docs.aws.amazon.com/vpc/latest/privatelink/create-endpoint-service.html).
+ - You have a global account and subaccount on SAP Business Technology Platform with SAP Private Link Service entitlement: [Set Up SAP Private Link Service on Amazon Web Services (Beta)](private-link-service-onboarding-aws).
+ - You have created an AWS Endpoint Service in the Amazon VPC Console. [Create a service powered by AWS Private Link](https://docs.aws.amazon.com/vpc/latest/privatelink/create-endpoint-service.html).
+ When creating the Endpoint Service, make sure you allowlist the SAP BTP CF AWS Account IDs as described in [Only allow requests from SAP BTP CF's AWS account](https://help.sap.com/docs/PRIVATE_LINK/42acd88cb4134ba2a7d3e0e62c9fe6cf/e0455888a6e44eb2bda8b8edb13dc55a.html?locale=en-US&version=CLOUD)
```
Comment (MichaelCzcz) [inline tutorials/private-link-aws/private-link-aws.md:17]: This should be a list item

File: tutorials/private-link-aws/private-link-aws.md [hunk 2]
```diff
@@ -9,19 +9,20 @@ primary_tag: software-product-function>sap-btp-cockpit
...
  - You have installed Cloud Foundry CLI. See [Install the Cloud Foundry Command Line Interface (CLI)](cp-cf-download-cli).
 
 ## You will learn
-  - How to create an SAP Private Link service instance to connect to your AWS Private Link Service using Cloud Foundry CLI.
+  - How to create an SAP Private Link instance to connect to your AWS Private Link Service using Cloud Foundry CLI.
```
Comment (MichaelCzcz) [inline tutorials/private-link-aws/private-link-aws.md:21]: Please remove the "." from all lines in this section.

File: tutorials/private-link-aws/private-link-aws.md [hunk 3]
```diff
@@ -158,7 +159,7 @@ To check the current status of the newly created service instance, you need the
 ### Bind application to service instance
 
 
-Upon the creation of a binding between a CF application and a private link service instance, SAP Private Link service creates a space-scoped [Cloud Foundry application security group](https://docs.cloudfoundry.org/concepts/asg.html) hat enables network access to the IP address associated with the private endpoint.
+Upon the creation of a binding between a CF application and a Private Link Service instance, SAP Private Link Service creates a space-scoped [Cloud Foundry application security group](https://docs.cloudfoundry.org/concepts/asg.html) hat enables network access to the IP address associated with the private endpoint.
```
Comment (MichaelCzcz) [inline tutorials/private-link-aws/private-link-aws.md:162]: Please fix typo (hat -> that)

File: tutorials/private-link-aws/private-link-aws.md [hunk 4]
```diff
@@ -202,4 +203,8 @@ After the creation of your service binding, your application receives the inform
 
 Congratulations! You have successfully completed the tutorial.
 
+[DONE]
```
Comment (MichaelCzcz) [inline tutorials/private-link-aws/private-link-aws.md:206]: Please remove this line.

File: tutorials/private-link-aws/private-link-aws.md [hunk 5]
```diff
@@ -202,4 +203,8 @@ After the creation of your service binding, your application receives the inform
...
+[ACCORDION-END]
```
Comment (MichaelCzcz) [inline tutorials/private-link-aws/private-link-aws.md:207]: Please remove this line.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

**Comment by MichaelCzcz** [comment]
Please resolve conflicts. To avoid conflicts, please always do a fetch+pull for your fork before pushing to production.

**Comment by jmmargo** [comment]
Thank you for making the revisions Madeline.  I did notice that this tutorial does not have a validation.  It is best practice to add at least one validation (knowledge check) to a tutorial.  I will merge this one, but I do recommend adding one in.

**Pull request merged**